### PR TITLE
p2 features integrated and minor bug fix

### PR DESCRIPTION
**Pull request by samyakshah3008**
p2 features added in tripleS Store now 

Live link: https://inspiring-almeida-52eb67.netlify.app/

### CODE REVIEW

File: styles/checkout/checkout.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+           <a href="/styles/ordersummary/ordersummary.html"><button class="signup-btn btn2">Proceed Payment</button></a> 
+
+        </div>
+
+
+     </main>
+
+</body>
+
+</html>
```
Comment (pritam-kr) [inline styles/checkout/checkout.html:104]: Remove this by breaking line

File: styles/product/product.css [hunk 2]
```diff
@@ -89,5 +89,33 @@
     width: 320px;
 }
 
+.page-btn{
+    display: flex;
+    gap: 1rem;
+    /* border: 2px solid black; */
```
Comment (pritam-kr) [inline styles/product/product.css:95]: remove comments

**Review by pritam-kr — APPROVED** [review]

**Review by lomesshh — APPROVED** [review]

File: styles/cart/cart.css [hunk 3]
```diff
@@ -58,3 +58,83 @@
     text-align: right;
 
 }
+
+.share-cart-btn{
+    /* border: 2px solid black; */
```
Comment (hinanshisuthar) [inline styles/cart/cart.css:63]: Remove comments which does not convey any information.

File: styles/cart/cart.css [hunk 4]
```diff
@@ -58,3 +58,83 @@
...
+    width: 150px;
```
Comment (hinanshisuthar) [inline styles/cart/cart.css:64]: Do you want this specific element to be of a certain width? If not prefer to use rems over px for values >=10 px

**Review by SJTGSHIVAM — COMMENTED** [review]
Great effort Samyak! looks good to me.

File: styles/checkout/checkout.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (hinanshisuthar) [inline styles/checkout/checkout.html:7]: Change the title to Checkout

**Comment by SJTGSHIVAM** [comment]
image
just a suggestion these buttons may look better at bottom and please change the tittle from document too. These are just small things rest everything is good.

File: styles/ordersummary/ordersummary.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+.green-title{
+    color: var(--success);
+}
```
Comment (hinanshisuthar) [inline styles/ordersummary/ordersummary.css:3]: Add a new line at the EOF. It is a good practice.

**Review by hinanshisuthar — COMMENTED** [review]
UI looks superb, Samyak. Well Done! Suggested some changes would be good if you implement them. Other than this, seems good to merge.

**Pull request merged**